src/components/ui/phoneAuth/index.tsx

```
@@ -1,109 +1,72 @@
+            callNumber: normalizedPhone
+          }
+        }
+      })
+
+      if (data?.sendMessage) {
+        success('인증코드가 문자로 발송되었습니다.')
+        setIsCodeSent(true)
+      } else {
+        error('인증코드 발송에 실패했습니다.')
+      }
+    } catch (e: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)